contentful_management/client.py

```diff
@@ -340,8 +340,16 @@ def _contentful_user_agent(self):
             'name': 'python',
             'version': platform.python_version()
         }
+
+        os_name = platform.system()
+        if os_name == 'Darwin':
+            os_name = 'macOS'
+        elif not os_name or os_name == 'Java':
+            os_name = 'Unknown'
```

Comment:
I hate to nitpick but there's no Unknown. If you can't identify it, just omit the section completely.